cli/options/config.go

```diff
@@ -0,0 +1,48 @@
+package options
```

Comment:
We need to move all configs to a separate package, maybe something like `neo-go/cli/config/`:
```
cli/config/
├── mainnet.go
├── mainnet_neofs.go
├── testnet.go
├── testnet_neofs.go
└── vm.go
...
```
These files should contain corresponding exported configurations. Then we need to create a separate CLI command (some `neo-go config generate`) that generates YAML config files in `neo-go/config` folder based on `neo-go/cli/config`. We also need to add a GA job that ensures that the content of `neo-go/config` matches the output of `neo-go config generate`.

Once it's implemented, we won't edit auto-generated YAML config files anymore. Instead, we will edit `cli/config` Go files and re-generate YAML configs.

@roman-khimov, ACK?

Reply:
Decided to embed our `config/protocol.*config.yml` files.